lib/ui/screens/settings/photo_cropped_result.dart

```diff
@@ -75,11 +75,10 @@ class _PhotoCroppedResultScreenState extends State<PhotoCroppedResultScreen> {
                           await registry.get<UserRepository>().updateUserPhoto(widget.imageBytes);
                       setState(() => _isUploading = false);
                       if (photoUploadResult) {
-                        showSnackBarSuccess(context, message: context.l10n.photo_changed_success);
-                        AutoRouter.of(context).pushAndPopUntil(
-                          EditPhotoRoute(imageBytes: widget.imageBytes),
-                          predicate: (route) => route.settings.name == UpdateProfileRoute.name,
-                        );
+                        context.pushRoute(EditPhotoRoute(
+                          imageBytes: widget.imageBytes,
+                          successMessage: context.l10n.photo_changed_success,
+                        ));
```

Comment:
```suggestion
                        AutoRouter.of(context).pushAndPopUntil(
                          EditPhotoRoute(
                            imageBytes: widget.imageBytes,
                            successMessage: context.l10n.photo_changed_success,
                          ),
                          predicate: (route) => route.settings.name == UpdateProfileRoute.name,
                        );
```

oficiálně SnackBar nejde nativně dát nahoru, protože Flutter followuje Material Design, který to "zakazuje" mít nahoře 👍


tady jen trošku problém, že po uložení se tahle route přidá do route stacku, tím pádem se může vrátit zpátky na tu confirmační screenu s fotkou (takže loop), ale správně by kliknutím na back button vrátit do hlavní editační menu screeny (tak jak to bylo předtím)

teď:
UpdateProfileRoute -> EditPhotoRoute -> CameraPhotoTakenRoute -> PhotoCroppedResultRoute -> EditPhotoRoute

předtím:
UpdateProfileRoute -> EditPhotoRoute -> CameraPhotoTakenRoute -> PhotoCroppedResultRoute 
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀^----------------------------------------------------|

Reply:
takže bych nechal toto, jen s tím přidaným argumentem `successMessage`:
```dart
                        AutoRouter.of(context).pushAndPopUntil(
                          EditPhotoRoute(
                            imageBytes: widget.imageBytes,
                            successMessage: context.l10n.photo_changed_success,
                          ),
                          predicate: (route) => route.settings.name == UpdateProfileRoute.name,
                        );
```

Reply:
Super, díky! ;-)
Neznám proces pull requestu, tak se omlouvám za blbou otázku... Je k tomuto požadavku ještě něco třeba ode mě? Předpokládám, že vy úpravu do kódu udělate na úrovni requestu a při případném mergy by již vaše změny měl být refelektované?

Reply:
já práva nemám, takže editovat nic nemůžu, ale tu to funguje tak, že ten kdo submitnul ten Pull Request, tak i ten ty suggested/requested změny udělá, a pak je pushne sem do PR :)

Reply:
jojo jak pise Matej, autor PRka dela pripadne upravy podle toho jak se domluvime v ramci code review

Reply:
Vrátil jsem původní routování viz řešení od @killalad. Díky!